test/goose/brokers/redis/batch_test.clj

```diff
@@ -1,17 +1,29 @@
 (ns goose.brokers.redis.batch-test
   (:require
+    [clojure.test :refer [deftest is testing use-fixtures]]
     [goose.brokers.redis.batch :as redis-batch]
     [goose.brokers.redis.commands :as redis-cmds]
     [goose.defaults :as d]
-    [goose.test-utils :as tu]
 
-    [clojure.test :refer [deftest is testing use-fixtures]]))
+    [goose.test-utils :as tu]))
 
 ;;; ======= Setup & Teardown ==========
 (use-fixtures :each tu/redis-fixture)
 
-;;; ======= TEST: Batch state creation ==========
+;;; ====================================
+
+(def opts {:redis-conn tu/redis-conn})
```

Comment:
Can we reuse `tu/redis-worker-opts` instead of redifining the opts here? During runtime, they'll be passed to the middleware.

Reply:
Correct. My bad.